scripts/vm-guest-tools.bat

```diff
@@ -22,7 +22,7 @@ if not exist "C:\Windows\Temp\windows.iso" (
 )
 
 cmd /c ""C:\Program Files\7-Zip\7z.exe" x "C:\Windows\Temp\windows.iso" -oC:\Windows\Temp\VMWare"
-cmd /c C:\Windows\Temp\VMWare\setup.exe /S /v"/qn REBOOT=R\"
+cmd /c C:\Windows\Temp\VMWare\setup64.exe /S /v"/qn REBOOT=R\"
```

Comment:
OK, never had a problem here as the setup.exe normally chooses the 64 bit installation. The upstream repo also uses setup.exe to allow the script run in 32bit templates eg. Win10. But if it's really a blocker for you I can change that.